development/print_db_data.py

```diff
@@ -122,50 +122,36 @@ def print_production_lines():
 def print_verbose_lines():
 	print('Data has been moved, this view is unavailable for now')
 	return
-	def _output_helper_prodlines(string, list):
```

Comment:
So that leaves the question: Is the code unnecessary? I understand you do not feel like fixing it, but maybe others would? Removing the old implementation entirely is going to make that much less likely.

Least you could do, in my opinion: open a ticket about this functionality (if there isn't one yet), include some sample output and a reference to the removed code there, and insert that ticket into the "has been moved" message line above.